lib/node_modules/@stdlib/stats/base/dists/rademacher/mean/benchmark/benchmark.js

```diff
@@ -0,0 +1,48 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var mean = require( './../lib' );
+var pkg = require( './../package.json' ).name;
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var i;
+	var y;
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		y = mean();
```

Comment:
Because of underlying implementation returning a constant, I would add a note along the lines of the following:

```suggestion
		// NOTE: certain compiler optimizations may render this benchmark meaningless...
		y = mean();
```

We have similar notes elsewhere when a function always returns the same value.